src/components/[guild]/Requirements/components/RequirementDisplayComponent.tsx

```diff
@@ -17,6 +17,14 @@ const RequirementDisplayComponent = ({
   rightElement,
   ...rest
 }: Props) => {
+  if ((requirement as any).isHidden) {
```

Comment:
If we add the `isHidden` prop to `RequirementType`, we can remove the typecasting here too. Or is there a reason why you decided not to add it to types? 👀

Reply:
No specific reason, I guess I just didn't want to add it because it's only used here, and not directly related to a requirement entity. But I tried it locally, and I think you are right, we should add it to the type instead, with a comment describing, that this is not really a part of a requirement